topi/python/topi/cuda/conv2d_transpose_nchw.py

```diff
@@ -186,7 +186,9 @@ def _callback(op):
 
             if cfg.is_fallback:
                 N, F, Y, X = get_const_tuple(conv.shape)
-                _fallback_schedule(N, F, Y, X)
+                # Workaround to make CUDA compilation work. Issue #4470
```

Comment:
Can we still use the fallback for 1x1 case by checking the input params here?

Reply:
I checked more kernel and strides combinations and found that the error happens when kernel is equal to strides, e.g.
```
# kernel and strides when compilation for CUDA fails
2x2 and (2,2)
3x3 and (3,3)
4x4 and (4,4)
5x5 and (5,5)
2x3 and (2,3)
3x2 and (3,2)
```

Reply:
added kernel / strides check and run `_fallback_schedule` for kernel 1x1 or when kernel != strides